src/validation_workflow/validation_args.py

```diff
@@ -43,25 +41,33 @@ def __init__(self) -> None:
             "--platform",
             type=str,
             choices=self.SUPPORTED_PLATFORMS,
-            help="Platform to validate.",
+            help="(optional) Platform to validate.",
             default="linux"
         )
         parser.add_argument(
             "--stgosbuild",
```

Comment:
We only have 1 build number. Prod does not have a build number as such here. To make it more clear we can just use `--os-build-number` and `osd-build-number` Keep the description as is it but stgosbuild does not sound right

Reply:
Thanks, @gaiksaya , sounds good, let us update it at next commit.